packages/server-wallet/src/wallet/index.ts

```diff
@@ -4,6 +4,8 @@ export {SingleChannelOutput, MultipleChannelOutput} from './types';
 import {MultiThreadedWallet} from './multi-threaded-wallet';
 import {SingleThreadedWallet} from './wallet';
 
+export {SingleThreadedWallet};
```

Comment:
https://api-extractor.com/pages/messages/ae-forgotten-export/

This means:
- we get a page for this class in our docs website. 
- we get a summary for this class in the api-report